hunts/src/AnswerCell.js

```diff
@@ -1,31 +1,103 @@
+import Badge from "react-bootstrap/Badge";
 import Button from "react-bootstrap/Button";
 import React from "react";
 import { updatePuzzle } from "./puzzlesSlice";
 import { useSelector, useDispatch } from "react-redux";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit, faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { showModal } from "./modalSlice";
 
 export default function AnswerCell({ row, value }) {
   const { id: huntId } = useSelector((state) => state.hunt);
   const dispatch = useDispatch();
-  return value ? (
-    <span className="text-monospace">{value}</span>
-  ) : (
-    <Button
-      variant="outline-primary"
-      onClick={() =>
-        dispatch(
-          showModal({
-            type: "SUBMIT_ANSWER",
-            props: {
-              huntId,
-              puzzleId: row.values.id,
-              puzzleName: row.values.name,
-            },
-          })
-        )
-      }
-    >
-      Submit Answer
-    </Button>
+  if (row.original.guesses === undefined || row.original.guesses.length == 0) {
+    return (
+      <Button
+        variant="outline-primary"
+        onClick={() =>
+          dispatch(
+            showModal({
+              type: "SUBMIT_ANSWER",
+              props: {
+                huntId,
+                puzzleId: row.values.id,
+                puzzleName: row.values.name,
+              },
+            })
+          )
+        }
+      >
+        Submit Answer
+      </Button>
+    );
+  }
+  return (
+    <>
+      {row.original.guesses.map(({ id, text }) => (
+        <>
+          <span className="text-monospace">{text}</span>
+          <span
+            style={{ cursor: "pointer" }}
+            onClick={() =>
+              dispatch(
+                showModal({
+                  type: "EDIT_ANSWER",
+                  props: {
+                    huntId,
+                    puzzleId: row.values.id,
+                    answerId: id,
+                    text,
+                  },
+                })
+              )
+            }
+          >
+            <Badge pill variant="light">
```

Comment:
lWhat do you think about just removing these `Badge`s around the edit/delete buttons, or making them less prominent (maybe `outline-light` instead of `light`)?

Reply:
I tried both -- didn't seem to make a huge difference.

Regardless, I think we should just remove these buttons (in a future PR) from the answer and name columns and have all of these options in a modal.

Reply:
Ok, let's merge this and then iterate on the UI afterwards